.idea/checkstyle-idea.xml

```diff
@@ -3,13 +3,10 @@
   <component name="CheckStyle-IDEA">
     <option name="configuration">
       <map>
-        <entry key="active-configuration" value="PROJECT_RELATIVE:$PROJECT_DIR$/.checkstyle/checkstyle.xml:EssentialsX" />
         <entry key="checkstyle-version" value="8.36.2" />
         <entry key="copy-libs" value="true" />
         <entry key="location-0" value="BUNDLED:(bundled):Sun Checks" />
         <entry key="location-1" value="BUNDLED:(bundled):Google Checks" />
-        <entry key="location-2" value="PROJECT_RELATIVE:$PROJECT_DIR$/.checkstyle/checkstyle.xml:EssentialsX" />
-        <entry key="property-2.checkstyle.suppressions.file" value=".checkstyle/suppressions.xml" />
```

Comment:
I'm not familiar with the checkstyle but why is this being removed?

Reply:
intellij is being lame